src/main/java/core/basesyntax/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.transaction.TransactionHandler;
+
+public class TransactionServiceImpl implements TransactionService {
+
```

Comment:
TransactionServiceImpl should contain Map<FruitTransaction.Operation, OperationHandler> transactionMap as class field and set this map via a constructor that gets 1 parameter  Map<FruitTransaction.Operation, Transaction>
```suggestion
    private Map<FruitTransaction.Operation, OperationHandler> operationMap;
    
    public TransactionServiceImpl(Map<FruitTransaction.Operation, OperationHandler> operationMap) {
        this. operationMap = operationMap;
    }
```